src/Workload/Microsoft.Maui.Controls.Sdk/Sdk/AutoImport.props

```diff
@@ -1,19 +1,14 @@
-<!-- all <ItemGroup/>'s must check if $(EnableDefaultMauiItems) is true -->
+<!-- every <ItemGroup/> must check the appropriate $(Enable*Items) properties -->
 <Project>
 
   <!-- Default .NET MAUI files-->
-  <ItemGroup Condition=" '$(EnableDefaultMauiItems)' == 'true' and '$(EnableDefaultEmbeddedResourceItems)' == 'true' ">
-    <MauiXaml Condition=" '$(EnableDefaultXamlItems)' == 'true' "  Include="**\*.xaml" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
-    <MauiCss  Condition=" '$(EnableDefaultCssItems)' == 'true' "   Include="**\*.css"  Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
+  <ItemGroup Condition=" '$(EnableDefaultMauiItems)' == 'true' ">
+    <MauiXaml Condition=" '$(EnableDefaultXamlItems)' == 'true' " Include="**\*.xaml" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
+    <MauiCss  Condition=" '$(EnableDefaultCssItems)' == 'true' "  Include="**\*.css"  Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
```

Comment:
Should we rename `$(EnableDefaultXamlItems)` and `$(EnableDefaultCssItems)`? Those came over from Forms.

Reply:
Yeah, they should probably have `Maui` in them? `$(EnableDefaultMauiXamlItems) and $(EnableDefaultMauiCssItems)`?

Reply:
Yes, we renamed the groups so we should rename the property I think to exactly what @mhutch has there.

Reply:
renmed